docs/changes.rst

```diff
@@ -1,6 +1,24 @@
 Wand Changelog
 ==============
 
+.. _changelog-0.6:
+
+0.6 series
```

Comment:
Is this up to date?

Reply:
Not update. The changelog usually gets updated after the changes pass regression testing.